src/java.base/unix/classes/sun/security/provider/NativePRNG.java

```diff
@@ -569,5 +653,13 @@ private void implNextBytes(byte[] data) {
                     throw new ProviderException("nextBytes() failed", e);
                 }
         }
+
+        /*[IF CRIU_SUPPORT]*/
+        private void clearRNGState() {
+            if (mixRandom != null) {
+                mixRandom.clearState();
+            }
         }
+        /*[ENDIF] CRIU_SUPPORT*/
+    }
```

Comment:
Please leave this line from upstream alone (with it's improper indentation).

Reply:
Updated

Reply:
I don't see a change; it should look like https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/master/src/java.base/unix/classes/sun/security/provider/NativePRNG.java#L572.

Reply:
Updated